docker-entrypoint.sh

```diff
@@ -43,6 +43,8 @@ if [ "$1" = 'postgres' ]; then
 		fi
 
 		{ echo; echo "host all all 0.0.0.0/0 $authMethod"; } >> "$PGDATA/pg_hba.conf"
+		{ echo; echo "host all all      ::/0 $authMethod"; } >> "$PGDATA/pg_hba.conf"
```

Comment:
It might be simpler to just use `all` for the address; then we only add one line to `hba.conf`.

> You can also write _all_ to match any IP address, _samehost_ to match any of the server's own IP addresses, or _samenet_ to match any address in any subnet that the server is directly connected to.
> https://www.postgresql.org/docs/9.3/static/auth-pg-hba-conf.html


(also, can you remove the extra blank line added below)